02/app.js

```diff
@@ -1 +1,24 @@
-console.log('DOM');
\ No newline at end of file
+console.log("DOM");
+
+const allLinksEls = document.querySelectorAll("[data-url]");
+console.log(allLinksEls);
+
+allLinksEls.forEach((link) => {
+	const urlLinks = link.getAttribute("data-url");
+	console.log(urlLinks);
+
+	link.setAttribute("href", urlLinks);
+});
+
+allLinksEls.forEach((link) => {
+	const hrefValue = link.getAttribute("href");
```

Comment:
Tą część zapewne można było zrobić w poprzednim `forEach` - nie ma co robić osobnej pętli jeśli nie trzeba bo to 2x więcej operacji ;)